Services/Concurrency/RateLimiting.cs

```diff
@@ -12,18 +12,20 @@ public interface IRateLimiting
         long GetPauseForNextTwinWrite();
         long GetPauseForNextMessage();
         double GetThroughputForMessages();
+        Models.Simulation.SimulationRateLimits GetRateLimits();
+        void SetCounters(Models.Simulation.SimulationRateLimits simulationRateLimits, ILogger log);
         void ResetCounters();
     }
 
     public class RateLimiting : IRateLimiting
     {
         // Use separate objects to reduce internal contentions in the lock statement
 
-        private readonly PerSecondCounter connections;
-        private readonly PerMinuteCounter registryOperations;
-        private readonly PerSecondCounter twinReads;
-        private readonly PerSecondCounter twinWrites;
-        private readonly PerSecondCounter messaging;
+        private PerSecondCounter connections;
```

Comment:
suggest renaming these to connectionsPerSecond, registryOperationsPerMinute, etc for consistency and clarity on the units.